manifests/install.pp

```diff
@@ -117,7 +117,7 @@
   } elsif ($::osfamily == 'RedHat') and ($package_version != 'present') {
     # Workaround PUP-5802/PUP-5025
     if ($::operatingsystem == 'Fedora') {
-      $pkg_os_suffix = 'fedoraf'
+      $pkg_os_suffix = 'fedora'
```

Comment:
Will that apply to just Puppet 5, or also future Puppet 4 releases? If this change means the module only works to upgrade to Puppet 5, I think we should work on supporting both names.